x-pack/legacy/plugins/ml/public/datavisualizer/file_based/components/import_settings/simple.js

```diff
@@ -50,6 +50,8 @@ export const SimpleSettings = injectI18n(function ({
         />
       </EuiFormRow>
 
+      <EuiSpacer size="m" />
```

Comment:
`EuiSpacer` is missing from the import section in this file https://github.com/elastic/kibana/blob/97d34690cf9eca802a65f06b9e69a03898288c1e/x-pack/legacy/plugins/ml/public/datavisualizer/file_based/components/import_settings/simple.js#L10

Reply:
Yep, that was my bad when I was reverting the prettier changes. @snide's on it